src/backend/dx11/src/lib.rs

```diff
@@ -1034,7 +1034,7 @@ impl RenderPassCache {
 
         cache
             .dirty_flag
-            .insert(DirtyStateFlag::GRAPHICS_PIPELINE | DirtyStateFlag::VIEWPORTS);
+            .insert(DirtyStateFlag::GRAPHICS_PIPELINE | DirtyStateFlag::PIPELINE_PS | DirtyStateFlag::VIEWPORTS);
```

Comment:
why does a subpass change `PIPELINE_PS` and not other stages?

Reply:
This change to the dirty flag is because the following clear_attachments invalidates VS, PS, etc